lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -39,7 +39,7 @@ def commands
               "search"     => "Searches module names and descriptions",
               "show"       => "Displays modules of a given type, or all modules",
               "use"        => "Selects a module by name",
-              "reload_lib"    => "Load a library file from specified path",
+              "reload_lib" => "Reload one or more library files from specified paths",
```

Comment:
I chose this wording carefully in order to suggest that reloading already loaded library files is the only supported operation.